backend/models/group.js

```diff
@@ -730,7 +731,9 @@ function buildRule(rule) {
 		throw responseCodes.INVALID_ARGUMENTS;
 	}
 
-	if (clauses.length > 1) {
+	if (clauses.length > 1 && notOperators.includes(rule.operator)) {
```

Comment:
@carmenfan I could also do `rule.operator.includes("NOT")` to detect this, but this would rely on the operator name.